redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+server:
+  port: 0
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
What about we add a default `app.name=redisSink` and the input default is `input: ${app.name}:input.  We should have this baked in s-c-s so we don't need to put it into the yml explicitly.